Hatch mech

benjamin-antupit opened the pull request.

benjamin-antupit committed:
Create turret subsystem 

Test for GitHub on mobile!

MajikalExplosions committed:
Adding turretturn

MajikalExplosions committed:
Finished up turret code

MajikalExplosions committed:
Added hatch code

It's probably really sketchy and I'm probably missing stuff. @yellowjaguar5 please check to make sure I didn't miss anything important?

benjamin-antupit committed:
some cleaning up.

Used lambdas instead of long method bodies, removed constructor, fixed whitespace, added comments, etc.

benjamin-antupit committed:
refactored Hatch as HatchClaw, moved enum to HatchClaw, added new ToggleMotor, separated hatch pneumatic from hatchClaw

benjamin-antupit committed:
Merge branch 'master' into hatch-mech

benjamin-antupit committed:
Merge branch 'master' into TurretSubsystem

benjamin-antupit committed:
Merge branch 'master' into hatch-mech

benjamin-antupit committed:
Merge branch 'master' into TurretSubsystem

benjamin-antupit committed:
whoops

benjamin-antupit committed:
added data to Shuffleboard

benjamin-antupit committed:
whitespace cleaning

benjamin-antupit committed:
Update ShuffleboardController.java

MajikalExplosions committed:
Merge branch 'master' into hatch-mech

MajikalExplosions committed:
Merge branch 'master' into TurretSubsystem

benjamin-antupit committed:
Merge branch 'master' into hatch-mech

benjamin-antupit committed:
Merge branch 'master' into TurretSubsystem

MajikalExplosions committed:
Added turret setpoint buttons

MajikalExplosions committed:
Added default clause

benjamin-antupit committed:
added/fixed comments/javadoc and added constructor

benjamin-antupit committed:
fixed POV button numbers

benjamin-antupit committed:
Update TalonWithMagneticEncoder.java

benjamin-antupit committed:
Merge branch 'master' into TurretSubsystem

benjamin-antupit committed:
moved counter to RobotMap since it actually has a port number (whoops)

MajikalExplosions committed:
Modified SetHatchPosition to toggle when it lacks a position argument.

MajikalExplosions committed:
Merge branch 'master' into TurretSubsystem

MajikalExplosions reviewed (approved):
@yellowjaguar5 pls review this too.

MajikalExplosions committed:
Merge branch 'hatch-mech' into TurretSubsystem

MajikalExplosions committed:
Fixed stuff

benjamin-antupit committed:
Update RobotMap.java

benjamin-antupit committed:
Update RobotMap.java

MajikalExplosions committed:
Merge pull request #28 from Gryphon-Robotics-4990/TurretSubsystem

Create turret subsystem

benjamin-antupit committed:
Make common methods in the DriveTrain subsystem static

benjamin-antupit committed:
Make common methods in HatchClaw static

benjamin-antupit committed:
Change InstantCommands from subclasses to methods that return an InstantCommand

benjamin-antupit committed:
Update robot port bindings

yellowjaguar5 commented:
@MajikalExplosions does everything look ready? I say go ahead.

benjamin-antupit committed:
Fixed TalonWithMagneticEncoder

Aux (second) selected sensor is not yet supported, so I moved all calls to the second sensor mode (absolute/Quadrature) to the (slower) SensorCollection class.

benjamin-antupit committed:
Update TalonWithMagneticEncoder.java

benjamin-antupit committed:
Update TalonWithMagneticEncoder.java
